testsuite/classlibrary/TestUnitTest.sc

```diff
@@ -44,7 +44,24 @@ TestUnitTest : UnitTest {
 		this.assertEquals( TestMixedBundleTester.findTestedClass, MixedBundleTester)
 	}
 
+	test_exception_assertException_error {
+		this.assertException({ 1789.monarchy }, DoesNotUnderstandError, "assertException should return true for any error")
+	}
```

Comment:
this test is no longer doing anything different than the `specificError` one below, so only one is needed at this point

Reply:
We know this fact only by our background knowledge of how errors are thrown. An explicit throw may be different from an error that just occurs.

Reply:
I see Julian's point, I suppose it can stay